Documentation/filters-strings.md

```diff
@@ -166,7 +166,9 @@ This filter accepts a parameter (boolean, default `false`) that controls the pre
 
 ## Filter: `swiftIdentifier`
 
-Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
+This filter has a couple of modes that you can specifiy using an optional argument (defaults to "normal"):
+
+**normal**: Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
 
 - Uppercase the first character.
```

Comment:
Oh, really, does that filter actually uppercase the first character all the time in our code (even since Swift 3)? Shouldn't by anymore I think… (if it's still the case, might be worth using that PR to make the `normal` mode correct again and stop uppercasing while there's not reason to?)